ios/versioned-react-native/ABI49_0_0/ReactNative/sdks/hermes-engine/hermes-utils.rb

```diff
@@ -73,7 +73,7 @@ def use_release_tarball(source, version, build_type)
 end
 
 def release_tarball_url(version, build_type)
-    return "https://repo1.maven.org/maven2/com/facebook/react/react-native-artifacts/#{version}/react-native-artifacts-#{version}-hermes-ios-#{build_type.to_s}.tar.gz"
+    return "https://github.com/expo/react-native/releases/download/sdk-49.0.0/ABI49_0_0hermes.tar.gz"
```

Comment:
thanks for the fix. would be good to update the versioning script here: https://github.com/expo/expo/blob/6f07fc9eac2eab7fdd58efc56d81e601af6621e5/tools/src/versioning/ios/transforms/podspecTransforms.ts#L107-L118